operations/service_deploy_smoke.go

```diff
@@ -264,29 +268,55 @@ func smokeTestEndpoints() cli.Command {
 				Usage: "file with test endpoints definitions",
 				Value: filepath.Join(wd, "scripts", "smoke_test.yml"),
 			},
+			cli.StringFlag{
+				Name:  projectNameFlagName,
+				Usage: "ID of project in which smoke test should run",
+				Value: "evergreen",
+			},
+			cli.StringFlag{
+				Name:  bvFlagName,
+				Usage: "build variant to run in the smoke test",
+				Value: "localhost",
+			},
+			cli.StringFlag{
+				Name:  cliPathFlagName,
+				Usage: "path to the Evergreen CLI to use for the smoke test",
+				Value: filepath.Join(wd, "clients", runtime.GOOS+"_"+runtime.GOARCH, "evergreen"),
+			},
+			cli.StringFlag{
+				Name:  cliConfigPathFlagName,
+				Usage: "path to the Evergreen CLI config to use for smoke test",
+				Value: filepath.Join(wd, "scripts", "agent-cli.yml"),
+			},
 			cli.StringFlag{
 				Name:  userNameFlagName,
-				Usage: "username to use with the API",
+				Usage: "username to use with the Evergreen API",
+				Value: "admin",
 			},
 			cli.StringFlag{
 				Name:  userKeyFlagName,
-				Usage: "key to use with the API",
+				Usage: "key to use with the Evergreen API",
+				Value: "abb623665fdbf368a1db980dde6ee0f0",
```

Comment:
Are there any concerns with this being in a public repo? Can this be used to run other evergreen operations by someone who shouldn't?

Reply:
This isn't a secret because [it's the publicly visible admin secret](https://github.com/evergreen-ci/evergreen/blob/1039fe8ff1a85cfb68346ab3df25faa929e84a90/testdata/smoke/users.json#L1) used to populate the smoke test app server's data. This isn't insecure unless you open up your local Evergreen servers to public traffic (which you should never do since local Evergreen is not secure).